features/polyExplorer/src/components/clusterStories/MatrixBubblesChart.css

```diff
@@ -22,3 +22,26 @@
     font-size: 14px;
     text-align: center;
 }
+
+.data-types-lengend {
+    display: flex;
+    margin: 24px 0;
+}
+
+.data-types-lengend .bubble-legend {
```

Comment:
I don't know if it's a typo here or here and in the class definition; worth checking it out anyway.

Reply:
Must be a typo

Reply:
Typo, I'll change it